__tests__/app.test.js

```diff
@@ -102,6 +103,54 @@ describe("GET /api/articles", () => {
   });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an array of all the comments according to the article Id provided", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toHaveLength(11);
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            body: expect.any(String),
+            votes: expect.any(Number),
+            author: expect.any(String),
+            article_id: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
+  test("200: Responds with an array of all the comments default sorted by most recent first", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+  test("400: sends an appropriate status and error message when given an invalid format id", () => {
+    return request(app)
+      .get("/api/articles/meltedbrain/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe(
+          "Bad request - article Id can only be a number"
+        );
+      });
+  });
+  test("404: sends an appropriate status and error message when given a valid but non-existent id for the article", () => {
+    return request(app)
+      .get("/api/articles/666/comments")
```

Comment:
again, maybe avoid `666` for your specific number (this is personal preference and opinion) - I use use 9000 or 9001 as an `over 9000` reference - a little bit jokey but also professional as it has no negative connotations. Completely up to you, just think about it

Reply:
I think you are completely right in this and the above comment, I have just edited both endpoints.